diff --git a/node_modules/react-router-nav-prompt/dist/useNavPrompt.js b/node_modules/react-router-nav-prompt/dist/useNavPrompt.js
index 6ed0196..b76813c 100644
--- a/node_modules/react-router-nav-prompt/dist/useNavPrompt.js
+++ b/node_modules/react-router-nav-prompt/dist/useNavPrompt.js
@@ -1,10 +1,10 @@
 import { useState, useRef, useEffect } from 'react';
-import { useHistory } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
 
 const useNavPrompt = ({
   shouldBlock
 }) => {
-  const history = useHistory();
+  const history = useNavigate();
 
   const _unblock = useRef(null);
 
@@ -34,8 +34,9 @@ const useNavPrompt = ({
 
       setBlocked(false);
 
+
       if (_navPath.current) {
-        history.push(_navPath.current);
+        history(_navPath.current);
       }
     }
   };
